### PR TITLE
doc: add usage for `uftrace info --symbol` with ELF binary directly.

### PR DESCRIPTION
**Pull request by ParkHanbum**
uftrace support viewing of symbol information from ELF binary. 
but there is no example from document. 
so, I'm add that!!

### CODE REVIEW

**Comment by namhyung** [comment]
Actually it was a hidden (undocumented) feature for debugging.  But maybe it'd be better having a documentation..

**Comment by namhyung** [comment]
Please make subject line shorter and add sign-off.

**Comment by ParkHanbum** [comment]
@namhyung  consider it done.

File: doc/uftrace-info.md [hunk 1]
```diff
@@ -78,6 +78,31 @@ To see the symbol table, one can use the `--symbols` option.
     [ 4] mcount (0x400570) size: 16
     [ 5] __cxa_atexit (0x400580) size: 16
 
+You can even use it without creating a record with the record option, like below:
```
Comment (namhyung) [inline doc/uftrace-info.md]: s/record option/symbols option/ ?

File: doc/uftrace-info.md [hunk 2]
```diff
@@ -78,6 +78,32 @@ To see the symbol table, one can use the `--symbols` option.
     [ 4] mcount (0x400570) size: 16
     [ 5] __cxa_atexit (0x400580) size: 16
 
+
+You can get symbol information from ELF binary directly without trace data that crafted by uftrace with record option.
```
Comment (namhyung) [inline doc/uftrace-info.md]: I think "that crafted by ..." part can be omitted.

Comment (ParkHanbum) [reply doc/uftrace-info.md]: @namhyung if you think so...

**Pull request merged**